intrinio/src/adapter.ts

```diff
@@ -47,7 +47,7 @@ export const startService = (config: Config): void => {
 }
 
 const customParams = {
-  base: ['base', 'from', 'asset'],
+  base: ['base', 'from', 'coin', 'market', 'asset'],
```

Comment:
'market' is not relevant here. When supplying params for a pair, the keys base, from, and coin are used together. Then quote, to, and market are used for the denomination.